cmake/openenclave-config.cmake.in

```diff
@@ -3,19 +3,27 @@
 
 @PACKAGE_INIT@
 
-include("${CMAKE_CURRENT_LIST_DIR}/openenclave-targets.cmake")
-
-set(OE_MAJOR "@PROJECT_VERSION_MAJOR@")
-set(OE_MINOR "@PROJECT_VERSION_MINOR@")
-set(OE_REVISION "@PROJECT_VERSION_PATCH@")
-set(OE_VERSION "@PROJECT_VERSION@")
+# Reference data.
 set(OE_BRANCH "@GIT_BRANCH@")
 set(OE_COMMIT "@GIT_COMMIT@")
 set(OE_BUILD_NUMBER "@BUILD_NUMBER@")
+
+set_and_check(OE_LIBDIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
```

Comment:
They are created by the package module [`configure_package_config_file`](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html) with this data below:

```
PATH_VARS CMAKE_INSTALL_LIBDIR CMAKE_INSTALL_BINDIR CMAKE_INSTALL_DATADIR CMAKE_INSTALL_INCLUDEDIR
```

This makes it relocatable 😄

Reply:
Trust me @CodeMonkeyLeet you probably don't want to read those docs. The relevant excerpt is:

> The variables <var1> to <varN> given as PATH_VARS are the variables which contain install destinations. For each of them the macro will create a helper variable PACKAGE_<var...>. These helper variables must be used in the FooConfig.cmake.in file for setting the installed location. They are calculated by configure_package_config_file so that they are always relative to the installed location of the package

and the rest explains how to do it properly. It was funny that we were using this module, but not actually using it 😅